docs/Developer Resources/Building ZFS.rst

```diff
@@ -5,24 +5,23 @@ GitHub Repositories
 ~~~~~~~~~~~~~~~~~~~
 
 The official source for ZFS on Linux is maintained at GitHub by the
```

Comment:
```suggestion
The official source for OpenZFS is maintained at GitHub by the
```

Reply:
Nice catch :+1: fixed another instance too in a followup commit